### PR TITLE
Design Updates

### PR DESCRIPTION
**Pull request by spbyrne**
Done:
- [X] Homepage design & content update.
- [X] Updated site header that includes sign in & sign up buttons.
- [X] Minor updates to blog & community pages
- [X] Docs header nav update

In Progress:
- [x] Homepage demo video & feature images
  - I've been working on fixing issues blocking my desired recording script. 
- [x] Decide on where we're pointing the two homepage CTAs

### CODE REVIEW

File: content/pages/home.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "blocks": [
     {
       "_template": "hero",
-      "headline": "Unlock your Next.js project with live editing",
-      "subline": "Empower your teammates to build and edit content stored in your Git repository.",
+      "headline": "Empower Your Team with Visual Editing",
+      "subline": "Tina allows teams to build faster and deliver to production with confidence.",
```
Comment (DirtyF) [inline content/pages/home.json:10]: When reading this out of context, one may wonder what are we building and delivering?

....to build **websites** faster and deliver **content** to production with confidence

File: components/DocumentationNavigation/DocsHeaderNav.tsx [hunk 2]
```diff
@@ -12,63 +13,106 @@ interface NavProps {
...
+                  <DynamicLink href={href} passHref>
+                    <a>{label}</a>
+                  </DynamicLink>
+                </li>
+              )
+            })}
+        </ul>
+        <div className="actions">
+          <ButtonGroup>
+            <Link href="https://auth.tina.io/">
```
Comment (DirtyF) [inline components/DocumentationNavigation/DocsHeaderNav.tsx:31]:
```suggestion
            <Link href="https://app.tina.io/signin">
```

File: components/DocumentationNavigation/DocsHeaderNav.tsx [hunk 3]
```diff
@@ -12,63 +13,106 @@ interface NavProps {
...
+              <Button size="small" color="blueInverted">
+                Sign In
+              </Button>
+            </Link>
+            <Link href="https://auth.tina.io/register">
```
Comment (DirtyF) [inline components/DocumentationNavigation/DocsHeaderNav.tsx:36]:
```suggestion
            <Link href="https://app.tina.io/register">
```

File: components/DocumentationNavigation/DocsHeaderNav.tsx (hunk 2 above)
Comment (DirtyF) [reply components/DocumentationNavigation/DocsHeaderNav.tsx:31]: Do we intend to publish it before the beta?

**Comment by spbyrne** [comment]
I put in my first stab at a new hero video but it's a bit rough and I think it can be improved; working on v2 now.

**Comment by scottgallant** [comment]
1. It's important that we go more narrow with the messaging above the fold.  That way, if you’re our specific niche persona (a Next.js website developer who can empathize with content creators), you’ll read this and know it’s for you. 

Instead of 

> Empower Your Team with Visual Editing
> Tina allows teams to build faster and deliver to production with confidence.

We should talk to that specific persona and mention Next or Git. My suggestion is something like this (I'm stealing @spbyrne's original headline).

> Next-gen Content Editing For Next.js Sites
> Tina brings a visual editing experience to content stored in your Git repository

2. In the 3rd section, "Dynamic" comes with a lot of history in web development. Can we update the heading from "Edit with Dynamic Visual Feedback" to "Edit with Real-time, Visual Feedback"

File: components/layout/CloudBanner.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  return (
+    <>
+      <div className="banner">
+        <div className="content">
+          <span className="desktop">
+            <p className="text">
+              <span className="wordmark">
+                <TinaCloudWordmark />
+              </span>{' '}
+              is in Alpha. Allow any team member to edit content!
```
Comment (DirtyF) [inline components/layout/CloudBanner.tsx:20]:
```suggestion
              is in Beta. Allow any team member to edit content!
```

File: content/pages/home.json [hunk 5]
```diff
@@ -4,47 +4,25 @@
...
-          "link": "/cloud",
-          "label": "Tina Cloud"
-        }
-      ]
-    },
     {
       "_template": "hero",
-      "headline": "Unlock your team with *next-gen editing*",
-      "subline": "Tina is an open-source editor for Jamstack sites",
+      "headline": "Empower Your Team with Visual Editing",
```
Comment (DirtyF) [inline content/pages/home.json:9]:
```suggestion
      "headline": "Visual Editing For Next.js Sites ",
```

File: content/pages/home.json (hunk 1 above)
Comment (DirtyF) [reply content/pages/home.json:10]:
```suggestion
      "subline": "Build visual editing on top of content versioned in GitHub",
```

File: content/pages/home.json [hunk 6]
```diff
@@ -53,46 +31,46 @@
...
           "media": {
-            "src": "/img/edit-alongside-content.gif"
+            "src": "/img/update-from-tina.jpg"
           }
         },
         {
           "_template": "feature",
-          "headline": "Built for the Jamstack",
-          "subline": "Tina is extensible by design, and already supports React and Next.js based websites.",
+          "headline": "Edit with Dynamic Visual Feedback",
```
Comment (DirtyF) [inline content/pages/home.json:42]:
```suggestion
          "headline": "Edit with Real-Time Visual Feedback",
```

**Comment by DirtyF** [comment]
New headline suggestion based on your comment in the blog post: https://tinacms-site-next-git-new-design-tinacms.vercel.app/

> Our goal is to 10x the experience for both developers and content editors — @scottgallant

File: components/layout/CloudBanner.tsx [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+            </p>
+          </span>
+          <Link href="/blog/announcement-tina-cloud-is-officially-in-alpha/">
+            <a className="link">
+              <span className="desktop">Read The Announcement</span>
+              <span className="mobile">Tina Cloud is in Alpha</span>
```
[the remark this answers is not in the document]
Comment (spbyrne) [reply components/layout/CloudBanner.tsx:26]: Good call given it's launch day now 😆  If you suggest changes for the blog link too I'll accept all

**Comment by logan-anderson** [comment]
>Query Your Content
Query your content across all your files using Tina Cloud’s GraphQL layer on top of the files stored in your repository.

> Command Line Quickstart
Bootstrap your Next.js project with a basic Tina setup and get started in minutes.

@spbyrne  It might be nice if these had links to their docs counterparts.

**Comment by logan-anderson** [comment]
Screen Shot 2021-08-18 at 9 09 59 AM

@spbyrne Are we changing this side bar? or is it just out of date? 

Maybe we need to merge main into this branch.

**Comment by perkinsjr** [comment]
@logan-anderson it's because this was created before the big change in the sidebar. The sidebar in tina.io will be the one.

**Comment by scottgallant** [comment]
@DirtyF good suggestions. I tweaked it a bit. See loom: https://www.loom.com/share/a07f0718b44d4b7286c6d3f9566ddec5

**Comment by scottgallant** [comment]
@spbyrne I'd love to see the following copy tweaks. Happy to discuss 

1. From "Share A Single Collaborative Workflow" to "Version Control Your Content" (something that speaks more directly to developers)
2. Change the descriptive text under "text in Build on Top of Your Components" to "Add an editing interface to your React components and give your content team the power to build complex pages." (previous version was a lot to grok)
3. From the Command Line Quickstart block, link to the "adding tina to next.js" guide 
4. The Command Line Quickstart screenshot needs to be updated. If you run that command you get an error. it should be two commands:
```
$ npx creat-next-app
$ npx @tinacms/cli init
```
5. Can we have those commands as text instead of a screenshot so people can copy and paste?

**Comment by DirtyF** [comment]
I'll implement your suggestions, to let Scott focus on assets.

File: components/layout/CloudBanner.tsx [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+        <div className="content">
+          <span className="desktop">
+            <p className="text">
+              <span className="wordmark">
+                <TinaCloudWordmark />
+              </span>{' '}
+              is in Alpha. Allow any team member to edit content!
+            </p>
+          </span>
+          <Link href="/blog/announcement-tina-cloud-is-officially-in-alpha/">
```
Comment (DirtyF) [inline components/layout/CloudBanner.tsx:23]:
```suggestion
          <Link href="/blog/tina-is-in-beta/">
```

File: content/pages/home.json [hunk 9]
```diff
@@ -4,95 +4,73 @@
...
-          "url": "https://tina.io/guides/tina-cloud/starter/overview/"
+          "url": "https://tina-tailwind-sidebar-demo-git-main-tinacms.vercel.app/"
         },
         {
           "variant": "ghost",
-          "label": "Try Demo",
+          "label": "Quickstart Guide",
           "icon": "",
-          "url": "https://tina-tailwind-sidebar-demo-git-main-tinacms.vercel.app/"
+          "url": "https://tina.io/guides/tina-cloud/starter/overview/"
```
Comment (DirtyF) [inline content/pages/home.json:22]:
```suggestion
          "url": "/guides/tina-cloud/starter/overview/"
```

File: pages/community.tsx [hunk 10]
```diff
@@ -142,29 +109,64 @@ function CommunityPage({ file: community, metadata, preview }) {
                     </InlineWysiwyg>
                   </InfoText>
                   <ButtonGroup>
-                    <DynamicLink
-                      href={'https://discord.gg/QC724mKx'}
-                      passHref
-                    >
-                      <Button as="a">Discord</Button>
+                    <DynamicLink href={'https://discord.gg/QC724mKx'} passHref>
```
Comment (DirtyF) [inline pages/community.tsx:112]:
```suggestion
                    <DynamicLink href={'https://discord.com/invite/zumN63Ybpf'} passHref>
```

**Comment by DirtyF** [comment]
Given we link to the blog post, this needs to be merged after #1006

**Comment by spbyrne** [comment]
I'll try to code the CLI screenshot so it can be easily updated / copied/pasted

**Comment by scottgallant** [comment]
@DirtyF , "Your" should be capitalized here: 

> Version Control your Content

**Comment by spbyrne** [comment]
Feature markdown support added so we can get links in there now

**Review by DirtyF — APPROVED** [review]

**Pull request merged**